### DISCUSSION TITLE
win 7 64bit

### DISCUSSION THREAD

**Discussion by m3vuv** [egzumer/uvk5-chirp-driver#23] [Q&A]
does it work with win7,i get all sorts of issues,ie it wont run  or load,is that normal?

**Comment by egzumer** [comment]
Windows 7 is an old, outdated system. I have no idea, I'm using windows 10 64bit and it works flawlessly.

**Reply by egzumer** [reply]
Firstly try running regular/original chirp, not my version. If it runs you should be able to load my driver there. If it doesn't run, search how to run chirp on win7. That is all I can help with this.

**Comment by joc2** [comment]
from chirp web site https://chirp.danplanet.com/projects/chirp/wiki/Download
just under the download link it's write this:

System requirements for CHIRP-next:   Windows 10 and later (64-bit)

![image](https://github.com/egzumer/uvk5-chirp-driver/assets/56229329/a127b688-4695-4cc0-b3da-2c90b4933574)
